### ISSUE TITLE
中英文测试集效果

### ISSUE THREAD

**Issue by rubby33** [neesetifa/bert_classification#2]
大佬，基于你blog中的中英文测试集。
我分别做了实验，发现用lstm效果挺差了：

英文：
2020-08-25 10:46:41,055 - long_text_classification.py[line:874] - INFO: ====== train: bert finetune Accuracy: 0.945222, Loss: 0.212253
2020-08-25 10:46:41,055 - long_text_classification.py[line:875] - INFO: ====== val: bert finetune Accuracy: 0.944176, Loss: 0.220052

2020-08-25 10:48:51,343 - long_text_classification.py[line:791] - INFO: Val Loss: 1.626610, val_acc:0.880488
2020-08-25 10:48:51,565 - long_text_classification.py[line:850] - INFO: test: lstm final  Accuracy: 0.899270, Loss: 1.617715

我用我自己的一个中文测试集：

2020-08-21 20:00:32,632 - long_text_classification_zh.py[line:929] - INFO: ====== eval: bert finetune Accuracy: 0.119825, Loss: 0.856958
2020-08-21 20:00:32,633 - long_text_classification_zh.py[line:930] - INFO: ====== test: bert finetune Accuracy: 0.117358, Loss: 0.883088

2020-08-21 20:08:05,607 - long_text_classification_zh.py[line:909] - INFO: val: lstm final  Accuracy: 0.838039, Loss: 1.071828
2020-08-21 20:08:05,608 - long_text_classification_zh.py[line:910] - INFO: test: lstm final Accuracy: 0.844314, Loss: 1.065375

求大佬指点

**Comment by tianzhaohaha** [comment]
> 大佬，基于你blog中的中英文测试集。 我分别做了实验，发现用lstm效果挺差了：
> 
> 英文： 2020-08-25 10:46:41,055 - long_text_classification.py[line:874] - INFO: ====== train: bert finetune Accuracy: 0.945222, Loss: 0.212253 2020-08-25 10:46:41,055 - long_text_classification.py[line:875] - INFO: ====== val: bert finetune Accuracy: 0.944176, Loss: 0.220052
> 
> 2020-08-25 10:48:51,343 - long_text_classification.py[line:791] - INFO: Val Loss: 1.626610, val_acc:0.880488 2020-08-25 10:48:51,565 - long_text_classification.py[line:850] - INFO: test: lstm final Accuracy: 0.899270, Loss: 1.617715
> 
> 我用我自己的一个中文测试集：
> 
> 2020-08-21 20:00:32,632 - long_text_classification_zh.py[line:929] - INFO: ====== eval: bert finetune Accuracy: 0.119825, Loss: 0.856958 2020-08-21 20:00:32,633 - long_text_classification_zh.py[line:930] - INFO: ====== test: bert finetune Accuracy: 0.117358, Loss: 0.883088
> 
> 2020-08-21 20:08:05,607 - long_text_classification_zh.py[line:909] - INFO: val: lstm final Accuracy: 0.838039, Loss: 1.071828 2020-08-21 20:08:05,608 - long_text_classification_zh.py[line:910] - INFO: test: lstm final Accuracy: 0.844314, Loss: 1.065375
> 
> 求大佬指点

你好，请问你知道这个模型里用的Bert模型在哪吗？
